docs/elements/hx-accordion-panel/index.html

```diff
@@ -41,13 +41,19 @@
 
 {% block attributes %}
   <dl>
+    <dt>disabled</dt>
+    <dd>Disables the panel</dd>
+
     <dt>open</dt>
     <dd>Opens the panel</dd>
   </dl>
 {% endblock %}
 
 {% block properties %}
   <dl>
+    <dt>disabled {Boolean [false]}</dt>
+    <dd>Determines if panel is disabled.</dd>
```

Comment:
What about:

```
Disables user interaction.
(i) Reflects the <code>disabled</code> attribute.
```

\* replacing `(i)` with the subdued subbody styling seen in tooltips or popovers.